cli/src/plugins/javadoc/buildIndexes.ts

```diff
@@ -39,4 +39,25 @@ export const buildIndexes = async ({
     path: "/index",
     root,
   });
+
+  const packages = finalizedProject.entities
+    .filter((e) => e.data?.category === "package")
+
+  packages.forEach(pkg => {
+    const pkgRoot = md.root([
+      toctree(finalizedProject.entities.filter((e) => e.data?.category === "class").filter((e) => e.data?.containingPackage === pkg.canonicalName)
+        .map(child => {
+          const sanitizedName = child.canonicalName.replace(/#.*$/, "");
+          const value = sanitizedName.substring(sanitizedName.lastIndexOf('.') + 1) // only want simple class name
+          return toctreeItem({
+            url: child.canonicalName.replace(/\./g,'/'),
+            value
+          })
+        }).filter((e) => e !== undefined) as ToctreeItemNode[])
+    ])
+    finalizedProject.writePage({
+      path: pkg.canonicalName.replace(/\./g,'/'),
```

Comment:
Style note: we tend to avoid abbreviations if it's not too burdensome (pkg -> package, pkgRoot -> packageRoot)

Reply:
package is a reserved word, it seems, so there isn't much choice in this case :(

Reply:
Womp!